posts/inside-rust/2019-10-11-AsyncAwait-Not-Send-Error-Improvements.md

```diff
@@ -121,9 +121,25 @@ async fn bar(x: &Mutex<u32>) {
 ```
 
 ```rust
-enum BarGenerator {
-    Unresumed { x: &Mutex<u32>, g: MutexGuard<'_, u32>, },
-    Suspend0 { x: &Mutex<u32>, g: MutexGuard<'_, u32>, }
+enum BarAsyncFnGenerator {
+    Unresumed {
+        // async fn params
```

Comment:
Could you keep the variant on a single line like it was before and move the comment to the line above? And could you change the comment to say:

```suggestion
        // `bar`'s parameters
```